### DISCUSSION TITLE
What is the unit of the frequency?

### DISCUSSION THREAD

**Discussion by ShefaaAbed** [ChampSim/ChampSim#480] [Q&A]
Hello, the frequency is 4000, but I need to know the unit, is it MHz?

"ooo_cpu": [
        {
            "frequency": 4000,
            "ifetch_buffer_size":64,
            "decode_buffer_size":32,
            "dispatch_buffer_size":32,
            "rob_size": 352,
            "lq_size": 128,
            "sq_size": 72,
            "fetch_width": 6,
            "decode_width": 6,
            "dispatch_width": 6,
            "execute_width": 4,
            "lq_width": 2,
            "sq_width": 2,
            "retire_width": 5,
            "mispredict_penalty": 1,
            "scheduler_size": 128,
            "decode_latency": 1,
            "dispatch_latency": 1,
            "schedule_latency": 0,
            "execute_latency": 0,
            "branch_predictor": "bimodal",
            "btb": "basic_btb"
        }]

**Comment by ngober** [comment]
Yes, that is correct. It is more clear in the source code, but perhaps not in the documentation.
